docs/development/release_management.rst

```diff
@@ -0,0 +1,122 @@
+Release Management
+==================
+
+The **Release Manager** is responsible for shepherding the release process to
+successful completion. This document describes their responsibilities. Some items
+must be done by people that have special privileges to do specific tasks
+(e.g. privileges to access the production apt server),
+but even if the **Release Manager** does not have those privileges, they should
+coordinate with the person that does to make sure the task is completed.
+
+Pre-Release
+-----------
+
+1. Open a **Release SecureDrop 0.x.y** issue to track release-related activity.
+   Keep this issue updated as you proceed through the release process for
+   transparency.
+2. Check if there is a new stable release of Tor that can be QAed and released
+   as part of the SecureDrop release. If so, file an issue.
+3. Check if a release candidate for the Tails release is prepared. If so, request
+   people participating in QA to use the latest release candidate.
+4. Prepare a changelog describing the changes in the release.
+5. Ensure that a pre-release announcement is prepared and shared with the community
+   for feedback. Once the announcement is ready, coordinate with other team members to
+   send them to current administrators, post on the SecureDrop blog, and tweet
+   out a link.
+6. For a regular release for version 0.x.0, branch off ``develop``:
+
+  ::
+
+     git checkout develop
+     git checkout -b release/0.x
+
```

Comment:
I would suggest prior to pushing to the branch, to ask a Freedomofpress GitHub org admin to enable branch protection